toolbelt/fd.h

```diff
@@ -194,7 +194,7 @@ class FileDescriptor {
   // same OS fd (that would be a mistake but there's no way to stop it).
   struct SharedData {
     SharedData() = default;
-    SharedData(int fd) : ref(1), fd(fd) {}
+    SharedData(int fd_) : ref(1), fd(fd_) {}
```

Comment:
What's the warning here?  Parameter with same name as a member?  Don't use trailing underscore for variables that aren't members.  Maybe just use 'f'?

Reply:
Same comment as on subspace PR, it's having the exact same name for parameter and member, see `-Wshadow-field-in-constructor`.

Reply:
Same response as for subspace, just don't use a trailing underscore, call it 'f' or something like that.

Reply:
Done.